src/renderers/dom/shared/eventPlugins/ChangeEventPlugin.js

```diff
@@ -289,6 +289,26 @@ function getTargetInstForInputOrChangeEvent(
   }
 }
 
+function handleControlledInputBlur(inst, node) {
+  // TODO: In IE, inst is occasionally null. Why?
```

Comment:
I threw a TODO in here. This gets called later down the road. I'm unclear why, in IE9/10 only, it's getting called with a nully instance.